addons/website_slides_survey/__init__.py

```diff
@@ -3,3 +3,11 @@
 
 from . import models
 from . import controllers
+
+from odoo import api, SUPERUSER_ID
+
+def uninstall_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    dt = env.ref('website_slides.badge_data_certification_goal', raise_if_not_found=False)
+    dt.domain = dt.domain.replace("('survey_scoring_success', '=', True)", "('completed', '=', True)") \
+        .replace("('slide_id.slide_category', '=', 'certification')", "(0, '=', 1)")
```

Comment:
I don't think much benefits of using `replace`. Just set the original value.
 
```suggestion
    if dt:
        dt.domain = "[('completed', '=', True), (0, '=', 1)]"
```